libpod/container_internal.go

```diff
@@ -683,9 +684,29 @@ func (c *Container) makeBindMounts() error {
 		c.state.BindMounts["/run/.containerenv"] = containerenvPath
 	}
 
+	// Add Secret Mounts
+	c.addSecrets()
+
 	return nil
 }
 
+// addSecrets mounts the secrets from the override and/or default mounts file
+func (c *Container) addSecrets() {
+	mountsFiles := []string{secrets.OverrideMountsFile, secrets.DefaultMountsFile}
+	for _, file := range mountsFiles {
+		secretMounts, err := secrets.SecretMounts(file, c.config.MountLabel, c.state.RunDir)
+		if err != nil {
+			logrus.Warn("error mounting secrets, skipping...")
+			continue
+		}
+		for _, mount := range secretMounts {
+			if _, ok := c.state.BindMounts[mount.Destination]; !ok {
```

Comment:
Does it matter that these will all be read-write instead of read-only?

Reply:
@rhatdan does it matter? I think buildah is also mounting them as rw. Will need to change that if they need to be ro.

Reply:
They are part of the container, so if the user inside the container wants to modify them, I don't see this as an issue.  Just like he can modify /etc/hosts and /etc/resolv.conf.